cluster/gce/config-default.sh

```diff
@@ -251,8 +251,17 @@ if [[ -n "${GCE_ALPHA_FEATURES:-}" ]]; then
 fi
 
 # Admission Controllers to invoke prior to persisting objects in cluster
-# If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
-ADMISSION_CONTROL=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota
+ADMISSION_CONTROL=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority
```

Comment:
define ENABLE_POD_SECURITY_POLICY first, then group the ADMISSION_CONTROL building together

Reply:
Removed.